expr/tagquery/meta_tag_record.go

```diff
@@ -33,13 +33,13 @@ func ParseMetaTagRecord(metaTags []string, queries []string) (MetaTagRecord, err
 // MatchesQueries compares another tag record's queries to this
 // one's queries. Returns true if they are equal, otherwise false.
 // It is assumed that all the queries are already sorted
-func (m *MetaTagRecord) MatchesQueries(other MetaTagRecord) bool {
-	if len(m.Queries) != len(other.Queries) {
+func (m *MetaTagRecord) MatchesQueries(other *MetaTagRecord) bool {
```

Comment:
KISS. just call this `Equals` and make the comments less redundant.

Reply:
Good idea

Reply:
Considering that this only compares the Expressions of the meta tag record, I think it would probably be better to name the method `EqualExpressions`. Otherwise that would be confusing if it's called `Equal` but it only compares a part of the meta tag record.